File: rest/test-credentials/test-sms-messages-example-2/test-sms-messages-example-2.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/user/account
        string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        string authToken = "your_auth_token";
        var client = new TwilioRestClient(accountSid, authToken);

        var message = client.SendMessage("+15005550006", "+15005550009", "Hey Mr Nugget, you the bomb!");
        Console.WriteLine(message.Sid);
    }
}

File: rest/test-credentials/test-sms-messages-example-2/test-sms-messages-example-2.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message = @client.account.messages.create(
  body: 'Hey Mr Nugget, you the bomb!',
  to: '+15005550009',
  from: '+15005550006'
)

puts message.sid
